### PR TITLE
Po ledger entries pdf links

### PR DESCRIPTION
**Pull request by lewisrenfrew**

### CODE REVIEW

File: src/Facade/ResourceBuilders/PurchaseOrderResourceBuilder.cs [hunk 1]
```diff
@@ -129,7 +130,13 @@ public PurchaseOrderResource Build(PurchaseOrder entity, IEnumerable<string> cla
                                             InvoiceRef = e.PlInvoiceRef,
                                             BaseVat = e.BaseVatTotal,
                                             InvoiceDate = e.InvoiceDate.ToString("dd/MM/yyyy"),
-                                            Tref = e.Pltref
+                                            Tref = e.Pltref,
+                                            Links = e.DocumentUploaded == "Y" 
+                                                        ? new LinkResource[1]
```
Comment (richardiphillips) [inline src/Facade/ResourceBuilders/PurchaseOrderResourceBuilder.cs:135]: do these need to be on the order resource? Won't most things that need it be requesting the purchase ledger entries anyway?

**Review by richardiphillips — APPROVED** [review]

File: src/Facade/ResourceBuilders/PurchaseOrderResourceBuilder.cs (hunk 1 above)
Comment (lewisrenfrew) [reply src/Facade/ResourceBuilders/PurchaseOrderResourceBuilder.cs:135]: the post invoices against orders screen fetches the order to render all of the data including the existing trefs posted against that order to stick in the table. It's in this table that I want to conditionally show the download button for a given tref it has a related file on the server. Do you think we should change that page to request the ledger entries from the finance endpoints as opposed to just using the order data?

Comment (richardiphillips) [reply src/Facade/ResourceBuilders/PurchaseOrderResourceBuilder.cs:135]: inv post gets the order for prices etc but requests the purchase ledger entries separately 
/ledgers/purchase?searchTerm={orderNumber} ?

Comment (lewisrenfrew) [reply src/Facade/ResourceBuilders/PurchaseOrderResourceBuilder.cs:135]: ah of course. I just saw the request for the order on that screen and naively assumed it was using that data in the table -  I'll just close this pr

**Pull request closed without merging**